src/Model/MessageManagerInterface.php

```diff
@@ -13,10 +13,10 @@
 
 namespace Sonata\NotificationBundle\Model;
 
-use Sonata\CoreBundle\Model\ManagerInterface;
-use Sonata\CoreBundle\Model\PageableManagerInterface;
+use Sonata\DatagridBundle\Pager\PageableInterface;
+use Sonata\Doctrine\Model\ManagerInterface;
 
-interface MessageManagerInterface extends ManagerInterface, PageableManagerInterface
+interface MessageManagerInterface extends ManagerInterface, PageableInterface
```

Comment:
This is BC?

Reply:
It is supposed to be, it's an alias.

Reply:
https://github.com/sonata-project/SonataCoreBundle/blob/3.x/src/CoreBundle/Model/ManagerInterface.php
https://github.com/sonata-project/SonataCoreBundle/blob/3.x/src/CoreBundle/Model/PageableManagerInterface.php